sdk/cosmos/azure-spring-data-cosmos/README.md

```diff
@@ -940,7 +940,7 @@ public class MultiDatabaseApplication implements CommandLineRunner {
 public class MultiTenantDBCosmosFactory extends CosmosFactory {
 
     private String tenantId;
-    
```

Comment:
Were these changes intended to be part of this PR?

Reply:
Reverted changes.